lib/Scheduler/bb_spill.cpp

```diff
@@ -386,18 +382,19 @@ InstCount BBWithSpill::CmputCost_(InstSchedule *sched, COST_COMP_MODE compMode,
   InstCount cycleNum;
   InstCount slotNum;
   SchedInstruction *inst;
-
+  
```

Comment:
Extra whitespace.

Reply:
Is SCF_SPILLS only meant to work with ACO?

Reply:
I am not exactly sure what SCF_SPILLS is meant to do. I believe it was implemented by Theodore a few years ago. I am uncertain if we still need it.

Reply:
I added a "simulated" register allocator. I think this was a try to use the simulated spills from that RA as a cost function. It can only be used with complete schedules I guess.

We should maybe have an assert if SCF_SPILLS and !CCM_STTC are enabled. Maybe that's a separate change this seems OK for now.